hera/README.md

```diff
@@ -30,6 +30,14 @@ git clone https://github.com/CosmosContracts/Juno
 make build && make install
 ```
 
+install dev tools 
+```
+sudo apt install make clang pkg-config libssl-dev build-essential git jq llvm libudev-dev -y
```

Comment:
Some of these are redundant with `build-essential`. Also, please use capitalization. :upside_down_face: